resources/wikia/modules/tracker.stub.js

```diff
@@ -88,7 +88,10 @@
 				TAKE_SURVEY: 'take-survey',
 
 				// View
-				VIEW: 'view'
+				VIEW: 'view',
+				
+				// View song lyrics (for cross-referencing against LyricFind reports) AN-861
+				VIEW_LYRIC: 'view-lyric'
```

Comment:
Maybe you could move this declaration to LyricWIki's tracking.js to keep the stub as clean as possible, it's used only in the tracking.js file anyway :)